components/Rank/__tests__/index.js

```diff
@@ -0,0 +1,27 @@
+import {mount} from 'enzyme'
+import React from 'react'
+import Rank from '../'
+
+describe('Rank', () => {
+  beforeEach(() => {
+    console.error = jest.fn((error) => {
+      throw new Error(error);
+    });
+  });
+
+  it('displays first place', () => {
+    expect(mount(<Rank ranking={1}/>).text()).toBe("1")
+  })
+
+  it('displays second place', () => {
+    expect(mount(<Rank ranking={2}/>).text()).toBe("2")
+  })
+
+  it('rejects missing ranking', () => {
+    expect(() => mount(<Rank/>)).toThrow()
+  })
+
+  it('rejects non-stringy rankings', () => {
+    expect(() => mount(<Rank ranking="dodgy"/>)).toThrow()
```

Comment:
If you're going to test that it throws an error for missing or wrong rankings, would it maybe also be worth testing what error it throws? I.e. would these two situations throw different errors? I didn't go down this route personally but wondering if you do think it is something worth testing that maybe it needs to be even clearer? (or am I talking nonsense?)